doc/lua/functions.md

```diff
@@ -379,6 +379,9 @@ rttr:GetWorld():AddAnimal(41, 44, SPEC_DUCK)
 
 [Back](#Lua-objects-and-their-methods)  
 
+**SetComputerBarrier(radius, x, y)**
```

Comment:
How do you undo this? The current implementation and the mission scripts allowed to remove the restriction

Reply:
1. You don't :) As mentioned earlier I didn't think this is supposed to mirror S2 behavior and  don't think the restrictions could be lifted in S2. Maybe @Spikeone can correct me on this. I think if more features (lifting, restricting for each player, etc.) are desired then they could be requested through an issue and with a separate PR. Unless they are considered a blocker for this PR of course.

2. If there is, it is not used in RttR. Basically everything is passed using built-in types, except enums.

Reply:
1. OK then.
2. I see. Then I'd just change the order to have the radius last to match the other functions of this class like `AddEnvObject(x, y, ...`

Reply:
> 1. You don't :) As mentioned earlier this is supposed to mirror S2 behavior and I don't think the restrictions could be lifted in S2. Maybe @Spikeone can correct me on this. I think if more features (lifting, restricting for each player, etc.) are desired then they could be requested through an issue and with a separate PR. Unless they are considered a blocker for this PR of course.
> 
>     2. If there is, it is not used in RttR. Basically everything is passed using built-in types, except enums.

Regarding 1., I'd really like to have that feature - but I do see, that it is currently not required or used at all. So this is no blocker for this PR I guess. Just asking (Since I don't know how this works) can't you just add a RemoveComputerBarrier with same parameters which just does the opposite?

Reply:
> can't you just add a RemoveComputerBarrier with same parameters which just does the opposite?

Currently not: What is stored are the affected points. There is no way to go backwards from that: Assume blocking 2 points close to each others with radius partially overlapping. If you want to remove one of the points you can't know if a currently blocked point belongs also to another points radius.

The alternative would be to store only the points and radius set, making them trivial to remove. In the check code the distance to each point would need to be calculated to decide if the currently checked point is blocked. This _might_ even be faster if there are few barrier points (those passed with radius) but with large radius.

Reply:
> Currently not: What is stored are the affected points. There is no way to go backwards from that: Assume blocking 2 points close to each others with radius partially overlapping. If you want to remove one of the points you can't know if a currently blocked point belongs also to another points radius.

I think, that this, for an initial implementation would be fine - so if you have overlapping areas, well, you may need to remove one and re-create the other again - thats fine for me